F2/genotyping.md

```diff
@@ -0,0 +1,142 @@
+
+---
+# Antimicrobial resistance (AMR) prediction
+
+Genotypic resistance can be predicted _in silico_ from whole genome sequencing (WGS) data. Successful _in silico_ prediction of AMR phenotype from WGS data is dependent upon sequence quality and having a comprehensive database of antimicrobial resistance genes (ARGs) and mutations conferring AMR.
+
+There are publicly available databases of AMR genes and point mutations, with varying levels of comprehensiveness, curation and regularity of updates. The method and database used influence the sensitivity and specificity of detection of ARGs and point mutations, and should be considered when comparing studies and outputs of surveillance programmes ([Doyle et al., 2020](https://pubmed.ncbi.nlm.nih.gov/32048983/)). Multiple studies have shown high concordance between AMR phenotype and genotype, including for _S. pneumoniae_, though specificity and sensitivity varies between antimicrobials ([Metcalf et al., 2016](https://www.sciencedirect.com/science/article/pii/S1198743X16303020); [Neuert et al., 2018](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5880904/); [Zankari et al., 2012](https://academic.oup.com/jac/article/68/4/771/705760?login=false)).
+
+It is important to remember that current _in silico _AMR prediction is not suitable for clinical purposes; they do not detect novel AMR mechanisms or variants with low homology to other genes in the database used, and require investigation with approaches such as genome wide association studies . Additionally, an ARG may be present but not expressed, resulting in non-concordance between genotype and phenotype ([Feldgarden et al., 2019](https://pubmed.ncbi.nlm.nih.gov/31427293/)).
+
+There are multiple available web-based and command-line tools for detecting antimicrobial resistance genes. The _S. pneumoniae _AMR gene library used by Pathogenwatch was provided by the GPS team and was last updated in April 2020. The tool works by using a Blast query of your assembly against the BLAST library of reference genes. A hit is called if the library sequence coverage (90%) and percentage identity (80%) threshold for a gene is met. For _S. pneumoniae_ Pathogenwatch predicts beta-lactam resistance, including penicillin resistance, using a machine learning model developed by the Centre for Disease Control and Prevention ([Li et al., 2016](https://pubmed.ncbi.nlm.nih.gov/27302760/); [Metcalf et al., 2016](https://www.sciencedirect.com/science/article/pii/S1198743X16303020)). This model generates minimum inhibitory concentration (MIC) predictions, and classification of isolates as susceptible, intermediate or resistant based on Clinical and Laboratory Standards Institute (CLSI) guidelines, including separate MIC thresholds for meningitis infections.
```

Comment:
There is currently a "_" before S. pneumoniae and _AMR in the second sentence. I presume the second "_" needs shifting left by one space to italicise S.pneumoniae